src/rewrite-helpers.jl

```diff
@@ -12,24 +12,15 @@ not the replacements.
 Set `fixpoint = true` to repeatedly apply rules until no
 change to the expression remains to be made.
 """
-function Base.replace(expr::Num, r::Pair, rules::Pair...)
-    _replace(unwrap(expr), r, rules...)
+function replacenode(expr::Num, r::Pair, rules::Pair...)
+    _replacenode(unwrap(expr), r, rules...)
 end
-
 # Fix ambiguity
-function Base.replace(expr::Num, rules...)
-    _replace(unwrap(expr), rules...)
-end
-
-function Base.replace(expr::Symbolic, rules...)
-    _replace(unwrap(expr), rules...)
-end
```

Comment:
I thought replace was fine though right? The occursin was the one which we couldn't define the right methods for? Either way, I'm fine with this change, but it would be good to add a deprection method for replace, I believe David Sanders uses it.

Reply:
I think that applies to occursin, not so much to replace.

Look for the next release that just got merged for the term interface fixes

Reply:
thanks

Reply:
should I update this PR to use `replace` instead of `replacenode`?

Reply:
`fixpoint` keyword arg is not in these top level methods, it should be.

Reply:
I'll update that as well?